packages/@glimmer/blueprint/files/package.json

```diff
@@ -7,33 +7,112 @@
+    "test:watch": "testem",
+    "lint": "eslint . --ext=js,ts"
   },
   "devDependencies": {
-    "@glimmer/application": "^1.0.0",
-    "@glimmer/application-pipeline": "^0.15.0",
-    "@glimmer/blueprint": "~<%= blueprintVersion %>",
+    "@babel/core": "^7.6.2",
+    "@babel/plugin-proposal-class-properties": "^7.5.5",
+    "@babel/plugin-proposal-decorators": "^7.4.4",
+    "@babel/preset-env": "^7.5.5",
+    "@babel/preset-typescript": "^7.3.3",
+    "@glimmer/babel-plugin-glimmer-env": "^1.0.0",
+    "@glimmer/babel-plugin-strict-template-precompile": "^1.0.0",
+    "@glimmer/blueprint": "~1.0.0",
     "@glimmer/component": "^1.0.0",
-    "@glimmer/inline-precompile": "^1.0.0",
-    "@glimmer/resolver": "^0.4.1",
-    "@glimmer/test-helpers": "^0.31.1",
+    "@glimmer/core": "^1.0.0",
+    "@glimmer/env": "^0.1.7",
+    "@glimmer/helper": "^1.0.0",
+    "@glimmer/modifier": "^1.0.0",
     "@types/qunit": "^2.0.31",
-    "broccoli-asset-rev": "^2.5.0",
-    "ember-cli": "^3.9.0",
-    "ember-cli-dependency-checker": "^2.0.1",
-    "ember-cli-inject-live-reload": "^1.6.1",
-    "tslint": "5.11.0",
-    "ember-template-lint": "^1.0.0-beta.6",
-    "ember-cli-uglify": "^2.0.0-beta.1",
-    "qunitjs": "^2.3.3",
-    "typescript": "~3.5.3"
+    "@typescript-eslint/eslint-plugin": "^2.2.0",
+    "@typescript-eslint/parser": "^2.2.0",
+    "babel-loader": "^8.0.6",
+    "clean-webpack-plugin": "^3.0.0",
+    "copy-webpack-plugin": "^5.1.1",
+    "css-loader": "^3.4.2",
+    "eslint": "^6.8.0",
+    "eslint-config-prettier": "^6.10.0",
+    "eslint-plugin-prettier": "^3.1.2",
+    "file-loader": "^5.0.2",
+    "glob": "7.1.6",
+    "html-webpack-plugin": "^3.2.0",
+    "qunit": "^2.9.3",
+    "qunit-dom": "^1.0.0",
+    "style-loader": "^1.1.3",
+    "testem": "^2.16.0",
+    "typescript": "~3.7.5",
+    "webpack": "^4.34.0",
+    "webpack-cli": "^3.3.4",
+    "webpack-dev-server": "^3.7.1"
+  },
+  "eslintConfig": {
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+      "ecmaVersion": 2019,
+      "sourceType": "module"
+    },
+    "env": {
+      "browser": true
+    },
+    "plugins": [
+      "@typescript-eslint",
+      "prettier"
+    ],
+    "extends": [
+      "eslint:recommended",
+      "prettier"
+    ],
+    "ignorePatterns": [
+      "dist/",
+      "node_modules/",
+      "!.*"
+    ],
+    "overrides": [
+      {
+        "files": [
+          "**/*.ts"
+        ],
+        "extends": [
+          "plugin:@typescript-eslint/eslint-recommended",
+          "plugin:@typescript-eslint/recommended",
+          "prettier/@typescript-eslint"
+        ]
+      },
+      {
+        "files": [
+          ".babelrc.js",
+          "testem.js",
+          "webpack.config.js"
+        ],
+        "env": {
+          "node": true
+        }
+      }
+    ]
+  },
+  "prettier": {
+    "singleQuote": true,
+    "trailingComma": "es5"
+  },
+  "browserslist": {
+    "production": [
+      "last 2 Edge versions",
```

Comment:
That’s a good question, we should check. My belief was that since Edge is evergreen, the “last 2” would continue to roll forward. I don’t think we should support the legacy edge once enough Chromium versions have been released.